### ISSUE TITLE
Figure out the RC branch flow

### ISSUE THREAD

**Issue by technophile-04** [scaffold-eth/create-eth#56]
### Description : 

The release candidate branch flow will come in handy when we have a lot of PR's at same time and we want to merge and publish an prerelease beta / rc version to npm before publishing the main version to npm. 

We kindof tried doing this in #53, but in #53 we manually published the beta version from local machine it would be nice if: 

- Merge all the PR to release candidate branch 
- as soon things are merge to that branch, similar to how "Version pacakges" is created to publish an rc version  which when merged publish an rc / beta version
- after a while we could also release a main version once the changes from release candidate branch is merged into main. 

We need to checkout if chageset provides some gh actions to help with this flow so that its smooth

**Comment by rin-st** [comment]
@technophile-04 can we close this?
